CHANGELOG.md

```diff
@@ -12,12 +12,15 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 - Adds a `contentId` prop to the `TemplateAppContainer` component and adds an `id` prop to the `TemplateContent` component. The default value of these `id`s are both set to "mainContent" and it will render as an attribute on the `main` HTML element. This is used as the target for the skip navigation link in consuming applications.
 - Adds an `Accessibility Guide` section to Storybook with a "Skip Navigation" page.
+- Adds `Reservoir` branding to Storybook.
```

Comment:
The new additions should be at the end of the list to keep the chronological order of when they were added into `development`.